decidim-admin/app/controllers/decidim/admin/participatory_process_user_roles_controller.rb

```diff
@@ -84,17 +83,16 @@ def resend_invitation
           end
         end
 
-        redirect_to participatory_process_user_roles_path(participatory_process)
+        redirect_to participatory_process_user_roles_path(current_participatory_process)
       end
 
       private
 
       def collection
-        @collection ||= ProcessAdminRolesForProcess.for(participatory_process)
-      end
-
-      def process_admin_roles
-        @process_admin_roles ||= ProcessAdminRolesForProcess.for(@participatory_process)
+        @collection ||= Decidim::ParticipatoryProcessUserRole
+                        .includes(:user)
+                        .where(participatory_process: current_participatory_process)
+                        .order(:role, "decidim_users.name")
```

Comment:
I removed it as a part of a huge clean up 😅 . I like these abstractions but I prefer using them when it's needed and since I managed to remove all occurrences but one I think it's better to keep it the logic in the same place.